pages/linux/qsub.md

```diff
@@ -0,0 +1,19 @@
+# qsub
+
+> Submits a script to the queue management system TORQUE on High Performance Computing clusters.
```

Comment:
Is there any reason why `High Performance Computing` is in title case? And is there any reason why a normal computing cluster couldn't use this tool? If not, then maybe something along the lines of `Often employed on high performance computing clusters.` would work as a separate sentence.

Reply:
I would rather remove subjective descriptions like "High Performance Computing Clusters", and just keep `Submits a script to the queue management system TORQUE`